Makefile

```diff
@@ -13,7 +13,7 @@ lint:
 	poetry run pyright
 
 test: clean
-	poetry run coverage run --omit=*/migrations/*,*answerking/* manage.py test --noinput
+	poetry run coverage run --omit=*/migrations/*,*answerking/*,*/tests/*,*manage.py manage.py test --noinput
```

Comment:
no test runs coverage on the code and this tells coverage not to analyse any file that has /tests/ in its path. We don't want to do this as coverage measures how much of the code is run when running tests. As such, test files always get 100% coverage. You can try it your self by running "make test" in develop. Because they always get 100% they skew the average coverage given at the end. That is why I ignored them.